python/lsst/phosim/utils/phosim_repackager.py

```diff
@@ -44,14 +46,23 @@ class PhoSimRepackager:
     MEFs with one HDU per amp, using LSE-400 for header conformity.
     """
 
-    TELCODE = 'MC'  # Telescope Code: MC is Main Camera
-    CONTRLLR = 'H'  # Controller: H for pHosim
+    CONTRLLR = "H"  # Controller: H for pHosim
```

Comment:
```
# Controller: H for PhoSim
```

Reply:
I'm emphasizing 'H' in the name to explain where it came from - same as https://github.com/lsst/obs_lsst/blob/801ecce4dfe125eb271c909e6cc86573ae5ee43e/python/lsst/obs/lsst/translators/lsst.py#L399  let me know if you would rather change that

Reply:
Maybe you could explain this in the comment. You could see if I have this doubt, other people might have this as well. Thanks for the explanation!

Reply:
ok, thank you!